src/features/defi/contexts/DefiManagerProvider/DefiCommon.ts

```diff
@@ -17,6 +18,44 @@ export enum DefiProvider {
   ThorchainSavers = 'THORChain Savers',
 }
 
+export const DefiProviderMetadata = {
+  [DefiProvider.Idle]: {
+    type: DefiProvider.Idle,
+    icon: IdleFinanceLogo,
+    name: 'Idle Finance',
+  },
+  [DefiProvider.Yearn]: {
+    type: DefiProvider.Yearn,
+    icon: 'https://rawcdn.githack.com/trustwallet/assets/master/blockchains/ethereum/assets/0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e/logo.png',
```

Comment:
get these from the asset service so they don't get out of sync

Reply:
Discussed this one out, those are provider icons, not asset icons, even though they happen to be the same here